### PR TITLE
fix: coingecko api duplicated data

### LINKED ISSUE #5006: [Bug] Deduplicate explorer in Trader Preview Card
## Bug Info

There to `etherscan.io`, only with different case addresses

image

image

### How to reproduce?

1.
2.
3.

### Mask Version:

- [ ] (Browser) Installed from the extension store (version: )
- [ ] (App) Installed from the app store (version: )
- [ ] master branch
- [ ] Build of version/commit hash:

(Please write down build command/flags if there is anything special)

### System

 
- [ ] Windows 10
- [ ] Mac OS X Catalina
- [ ] Linux: 

### Platform/Browser

 
- [ ] Latest Chrome (stable channel)
- [ ] Latest Firefox (stable channel)
- [ ] Android 11
- [ ] iOS 14

### PR DESCRIPTION
**Pull request by zhouhanseng**
closes #5006

image

### CODE REVIEW

**Comment by zhouhanseng** [comment]
@DimensionDev/qa

File: packages/mask/src/plugins/Trader/apis/coingecko/index.ts [hunk 1]
```diff
@@ -113,7 +114,9 @@ export async function getCoinInfo(coinId: string) {
         `${COIN_GECKO_BASE_URL}/coins/${coinId}?developer_data=false&community_data=false&tickers=true`,
         { cache: Flags.trader_all_api_cached_enabled ? 'force-cache' : 'default' },
     )
-    return response.json() as Promise<CoinInfo>
+    const data = (await response.json()) as CoinInfo
+    data.links.blockchain_site = uniq(data.links.blockchain_site.map((url) => url.toLowerCase()))
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Trader/apis/coingecko/index.ts:118]: Move this to `getCoinTrending` in `packages/mask/src/plugins/Trader/apis/trending/index.ts`.

File: packages/mask/src/plugins/Trader/apis/trending/index.ts [hunk 2]
```diff
@@ -209,7 +209,9 @@ async function getCoinTrending(id: string, currency: Currency, dataProvider: Dat
                     ].filter(Boolean),
                     source_code_urls: Object.values(info.links.repos_url).flatMap((x) => x),
                     home_urls: info.links.homepage.filter(Boolean),
-                    blockchain_urls: [platform_url, ...info.links.blockchain_site].filter(Boolean),
+                    blockchain_urls: uniq(
+                        [platform_url, ...info.links.blockchain_site].filter(Boolean).map((url) => url.toLowerCase()),
```
Comment (guanbinrui) [inline packages/mask/src/plugins/Trader/apis/trending/index.ts:213]: 👍

**Review by guanbinrui — APPROVED** [review]

**Comment by AinuX** [comment]
Checked

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**